README.md

```diff
@@ -1,6 +1,6 @@
 # openfunction
 
-![Version: 0.3.1](https://img.shields.io/badge/Version-0.3.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.8.0](https://img.shields.io/badge/AppVersion-0.8.0-informational?style=flat-square)
+![Version: 0.3.2-rc.0](https://img.shields.io/badge/Version-0.3.2--rc.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.8.1-rc.0](https://img.shields.io/badge/AppVersion-0.8.1--rc.0-informational?style=flat-square)
```

Comment:
Maybe we can use `0.4.0` directly instead of `0.3.2-rc.0`.

Reply:
The rc version is required ? corresponding to OpenFunction v0.8.0-rc.0

Reply:
ok then maybe 0.4.0-rc.0 ?

Reply:
ok